FRPs/proposal-2-liquidations.md

```diff
@@ -0,0 +1,50 @@
+
+Drawing the link to MEV, each liquidation event may qualify as Miner Extractable Value (MEV). For example, in a recent price oracle manipulation attack, a liquidator performed an oracle price post and liquidated millions of USD within one transaction, yielding a profit of nearly 4M USD. 
+
+The goal of this FRP is to offer a systematization of existing liquidation mechanisms and consistent measurements of the past and current DeFi liquidation state. We aim to investigate alternative liquidation designs in an effort to mitigate MEV from liquidation events.
+
+## Background and Problem Statement
+
+The goal of this FRP is to study the past and current status of blockchain liquidation. Specifically, we would like to answer the following questions:
+- What is the monthly accumulated liquidation profit for Aave, Compound, dYdX, MakerDAO?
+- What gas price distributions are being paid for the liquidation events?
+- What are the different liquidation methods, and how do they differ with respect to MEV? Which liquidation mechanism avoids MEV?
+- How sensitive are current debts to liquidation (by how much would an adversary need to manipulate a price oracle)? An MEV miner could try to manipulate an oracle to extract this MEV.
+- How can we design a possible liquidation mechanism that mitigates MEV?
+
+## Plan and Deliverables
+
+- Our report is envisioned to contain multiple figures/quantitative results regarding: 
+  - Monthly liquidation profits for MakerDAO/Aave/Compound/dYdX.
+  - Understand the gas prices of liquidation events. Is there a way to avoid gas fee bidding contests in liquidations in an effort to preserve valuable block-space?
+  - Quantify the liquidation sensitivity (how much collateral is liquidated given a % price decline of the collateral) for MakerDAO/Aave/Compound/dYdX and most of the assets on these platforms.
+  - How many liquidation events are profitable to the liquidator?
+  - How many liquidations make use of flash loans?
+ 
+- We plan to systematize the different liquidation mechanisms in an effort to understand their strengths and weaknesses.
+- We would love to integrate with the inspectors of the flashbots dashboard to crawl our data. Yet, we haven’t been able to look into further technical details of how they work and what the current architecture is. If too complicated we can also proceed with our current geth-based infrastructure.
+- We plan to design a new liquidation mechanism that would prevent or mitigate MEV. There may be an inherent tension between a minimal MEV that must remain for the system to sustain, and we would like to better understand this tension.
```

Comment:
Do you have any intuition around how you'd do this?

Reply:
Not sure yet, but there intuitively should be a way.

Reply:
To me, studying this fundamental trade-off is the most interesting part of this proposal.  Can we generalize this more?  How can we make sure the artifact/work survives the completion of the proposal?

Reply:
Agreed, I think while designing a new liquidation mechanism we hopefully will understand more about this tradeoff. By generalizing, do you mean on how to design protocols in general to minimize MEV?